tests/installation/installation_overview.pm

```diff
@@ -119,8 +119,11 @@ sub run {
         # select None for Major Linux Security Module.
         set_linux_security_to_none if (is_sle('>=15-SP4') && check_screen("apparmor-not-selected") && !(get_var('PATTERNS') =~ 'default|all|apparmor'));
         ensure_ssh_unblocked;
-        $self->check_default_target();
+        until (!check_screen("install-overview-options-evaluating-pkg-selection")) {
+            save_screenshot;
+        }
     }
+    $self->check_default_target();
```

Comment:
I don't understand what you are trying to achieve but I am pretty sure this will break a lot of tests. I suggest you consult with the test module maintainers and also I suggest to run an extensive set of test scenarios. Without a timeout this will run into MAX_JOB_TIME if the needle tag install-overview-options-evaluating-pkg-selection is never matched. Also because `check_screen` checks once without waiting time this will be a busy-loop occupying the system and wasting CPU ressources until a match is found so also discouraged.

Reply:
Hello,Oliver. Because we encountered some machines' job failure during installation.The time of evaluate packages in the installation overview phase is relatively long, which will lead to failure, so these two lines are added. See details:http://10.67.129.4/tests/56464#

Reply:
I updated the commit comments.

Reply:
I think you confused the github description with git commit message. The git commit message is still just one line

Reply:
Thank you for giving me so much advice and knowledge. I will study the knowledge you shared about git again.
Please help me check again. I am sorry that this PR wasted your so much time.